.github/workflows/release.yml

```diff
@@ -97,3 +94,113 @@ jobs:
           ${{steps.get-checksums-from-draft-release.outputs.result}}
           EOF
           )
+
+      - name: generate homebrew formula
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          set -e -x
+          VERSION=`echo ${{ github.ref }}  | grep -Eo '[0-9].*'`
+
+          IMGPKG_DARWIN_AMD64=$(grep 'imgpkg-darwin-amd64' <<< "${{steps.get-checksums-from-draft-release.outputs.result}}" | awk '{print $1}')
+          IMGPKG_LINUX_AMD64=$(grep 'imgpkg-linux-amd64' <<< "${{steps.get-checksums-from-draft-release.outputs.result}}" | awk '{print $1}')
+
+          cat <<EOF > ./dist/imgpkg.rb
+          class Imgpkg < Formula
+            desc "Imgpkg"
+            homepage "https://carvel.dev/imgpkg/"
+            version "v${VERSION}"
+
+            on_macos do
+              url "https://github.com/vmware-tanzu/carvel-imgpkg/releases/download/v${VERSION}/imgpkg-darwin-amd64"
+              sha256 "$IMGPKG_DARWIN_AMD64"
+            end
+
+            elsif OS.linux?
```

Comment:
this elif doesnt look right...

Reply:
hmmm how so?

https://github.com/vmware-tanzu/homebrew-carvel/blob/develop/imgpkg.rb#L9